docs/issuer/issuer-overview.md

```diff
@@ -83,3 +84,34 @@ Issuer Node API UI with Managing Schemas, Credentials and other functionalities:
 <div align="center">
 <img src= {useBaseUrl("img/3002.png")} align="center" />
 </div>
+
+### Privado Identity Chain
+
+The Privado Identity Chain, currently in beta with limited functionality, is a blockchain network designed to serve as the primary chain for user identities. It will act as the authoritative source of truth, enabling other chains to synchronize their state with it. This chain is essential for preventing the fragmentation of identities and credentials across the various chains we will be working with, ensuring that identities and credentials remain consistent and reusable across all chains.
+
+DIDs associated with the Privado Identity Chain will follow a prefix format:
+<Tabs>
+<TabItem value="Mainnet">
+
+```main
+did:iden3:privado:main:
+```
+</TabItem>
+<TabItem value="Testnet">
+
+```test
+did:iden3:privado:test:
+```
+</TabItem>
+</Tabs>
+For example, a DID would look like this:
+
+did:iden3:privado:main:2ScrbEuw9jLXMapW3DELXBbDco5EURzJZRN1tYj7L7
+
+:::note
+State transitions and MTP issuance are not currently supported.
+:::
+
+:::caution
+Currently, a few issuers, such as Synaps, are deployed on the Privado Identity Chain. During the beta phase, where the chain has limited functionality, issuers can continue to use other supported chains, such as Polygon POS, until the full version of the Privado Identity Chain is released. 
```

Comment:
Add that then issuers both on other chains and privado would be supported. Because otherwise it sounds like issuers on other chains will be not supported as soon as we release privado chain.

Reply:
This is fixed